src/TimeGutter.jsx

```diff
@@ -1,39 +1,53 @@
 import React from 'react';
-import cn from 'classnames';
 import dates from './utils/dates';
-import localizer from './localizer'
+
+import TimeSliceGroup from './TimeSliceGroup.jsx'
 
 let TimeGutter = React.createClass({
 
   propTypes: {
     step: React.PropTypes.number.isRequired,
+    slices: React.PropTypes.number,
+    now: React.PropTypes.instanceOf(Date).isRequired,
     min: React.PropTypes.instanceOf(Date).isRequired,
-    max: React.PropTypes.instanceOf(Date).isRequired
+    max: React.PropTypes.instanceOf(Date).isRequired,
+    showlabels: React.PropTypes.bool,
+    timesliceClassnames: React.PropTypes.string,
+    culture: React.PropTypes.string,
+    timeGutterFormat: React.PropTypes.string
+  },
+
+  getDefaultProps() {
+    return {
+      step: 10,
+      slices: 2,
+      now: new Date(),
+      showlabels: true,
+      culture: 'en'
+    }
+  },
+
+  renderTimeSliceGroup(key, isNow, date) {
+    return <TimeSliceGroup key={key} isNow={isNow} slices={this.props.slices} showlabels={this.props.showlabels}
```

Comment:
for consistency, if components are multi-line can we do 1 prop per line and surround the whole component in `( )`